lib/registries/addon/drafts/draft/route.ts

```diff
@@ -68,6 +71,21 @@ export default class DraftRegistrationRoute extends Route {
         };
     }
 
+    @action
+    willTransition(transition: Transition) {
```

Comment:
Rewrite this using `Router.on('routeWillChange')`
https://deprecations.emberjs.com/v3.x/#toc_deprecate-router-events h/t @brianjgeiger